github/Project.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
+
+############################ Copyrights and license ############################
+#                                                                              #
+# Copyright 2018 bbi-yggy <yossarian@blackbirdinteractive.com>                 #
+#                                                                              #
+# This file is part of PyGithub.                                               #
+# http://pygithub.readthedocs.io/                                              #
+#                                                                              #
+# PyGithub is free software: you can redistribute it and/or modify it under    #
+# the terms of the GNU Lesser General Public License as published by the Free  #
+# Software Foundation, either version 3 of the License, or (at your option)    #
+# any later version.                                                           #
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+################################################################################
+
+import datetime
+import json
+
+import github.GithubObject
+import github.ProjectColumn
+
+# Original request for project support:
+#   https://github.com/PyGithub/PyGithub/issues/606
+# and pull request with initial functionality:
+#   https://github.com/PyGithub/PyGithub/pull/854
+
+# 'Accept' header required while Github Projects API still in preview mode.
+PROJECT_PREVIEW_HEADERS = {"Accept": "application/vnd.github.inertia-preview+json"}
+
+class Project(github.GithubObject.CompletableGithubObject):
```

Comment:
I made this guy "completable" but to be honest I don't know what that means.

Reply:
Your guess is correct.